lib/lcdbwf/R/functional_enrichment.R

```diff
@@ -1,3 +1,106 @@
+#' Function to run enrichment on a res_list
```

Comment:
@mitraak, a lot of the code elsewhere has changed since this PR, making it a pretty messy diff. Can you pull out just this parallel part, make a branch off of `v1.10` to add it to, and open a new PR targeting `v1.10`?